### DISCUSSION TITLE
Delete old Connections (Tabular Server)

### DISCUSSION THREAD

**Discussion by RainerZufall1** [DaxStudio/DaxStudio#762] [Q&A] [answered]
Hi,

short Question.
How can I delete old / incorrect entries in the connection window? 

![image](https://user-images.githubusercontent.com/93875307/142174748-a13329b4-4399-4fba-8179-4c90dde653fb.png)

kind regards

**Comment by dgosbell** [comment, marked as answer]
That's a good question. We don't actually have any way to do this in the User Interface at the moment. I've created a new feature request here to track this as an idea https://github.com/DaxStudio/DaxStudio/issues/763 

If you don't mind making changes using regedit the list of recent servers is kept in the registry at the following location `Computer\HKEY_CURRENT_USER\SOFTWARE\DaxStudio\ServerMRU` and it is safe to just delete one or more of those entries (you don't need to rename the keys of the existing entries or anything like that)

**Reply by RainerZufall1** [reply]
Perfect
Thanks a lot :)
